docs/_specification/1.2-DRAFT/appendix/implementation-notes.md

```diff
@@ -46,7 +46,15 @@ When implementing tools to work with RO-Crate it is not necessary to use JSON-LD
 
 ## Combining with other packaging schemes
 
-RO-Crates may co-exist with other packaging schemes, such as [BagIt] using two general approaches; either (a) _adding_ RO-Crate into a package as part of the payload or (b) _wrapping_ another kind of package. Examples using BagIt follow.
+RO-Crates may co-exist with other packaging schemes, such as [BagIt] or [ELN] using two general approaches; either (a) _adding_ RO-Crate into a package as part of the payload or (b) _wrapping_ another kind of package.
```

Comment:
Would be good to include what ELN stands for - is this the right expansion of the name?
```suggestion
RO-Crates may co-exist with other packaging schemes, such as [BagIt] or [ELN] (Electronic Lab Notebook) using two general approaches; either (a) _adding_ RO-Crate into a package as part of the payload or (b) _wrapping_ another kind of package.
```